sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -518,6 +519,11 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class RedirectPolicyOptions
```

Comment:
Is this analogous to RetryOptions?  If it is, would it make sense to drop `Policy` from the name?  Or is communicating that these only apply to the policy important?  It doesn't look like we have `PolicyOptions` elsewhere in the Core API, so this feels like introducing a new pattern we might want to adhere to in the future ... is that the goal?

Reply:
Initially this was named relevant to the transport, however since this is only affecting the behavior of the RedirectPolicy itself, @KrzysztofCwalina suggested we scope the name to that to be explicit about what is being affected.

Reply:
We could call it ClientRedirectsOptions